bundles/org.openhab.core/src/main/java/org/openhab/core/internal/items/MetadataRegistryImpl.java

```diff
@@ -61,6 +64,18 @@ public boolean isInternalNamespace(String namespace) {
         return namespace.startsWith(INTERNAL_NAMESPACE_PREFIX);
     }
 
+    /**
+     * Provides all namespaces of a particular item
+     *
+     * @param itemname the name of the item for which the namespaces should be searched.
+     */
+    @Override
+    public Set<String> getAllNamespaces(String itemname) {
+        return getAll().stream().filter(n -> {
+            return n.getUID().getItemName().equals(itemname);
+        }).map(n -> n.getUID().getNamespace()).collect(Collectors.toSet());
```

Comment:
```suggestion
        return getAll().stream().filter(n -> n.getUID().getItemName().equals(itemname))
            .map(n -> n.getUID().getNamespace()).collect(Collectors.toSet());
```

Reply:
good catch, I had the "return" there during development and forgot to take it out.